code/modules/shuttle/shuttle.dm

```diff
@@ -552,6 +552,21 @@
 	else
 		. = null
 
+/obj/docking_port/mobile/proc/check_exile_pass()
```

Comment:
Also btw wont this return false if a shuttle isnt on station? Is this tested?

Reply:
Looks like I removed the return TRUE that was supposed to follow that code. 

Thanks for pointing that out.